Punt37/Form1.cs

```diff
@@ -5,9 +5,12 @@ namespace Punt37
 {
     public partial class Main : Form
     {
-        public Main()
+        private bool useHttps;
+
+        public Main(bool useHttps)
```

Comment:
I realize it is super ad hoc to let this argument be a boolean. Let me know if you think it's better to let the argument be something more generic, perhaps to support future expansion of options. Same for `parseArgs()` below.

Reply:
I'd prefer to make HTTPS default.  I didn't set this to HTTPS originally because in the past I've seen lack of SSL certs as a stopping point.  You've tested and this works from Euphonia Shortcuts as is?  Do you turn off SSL cert validation and does this stack auto-generate the appropriate SSL signed certificate?

Reply:
Yes, we turn off cert validation at the client side. The code for generating the certificate is separate from this code base.

Reply:
My suggestion is to keep http as the default and let https be an option that can be activated by the command-line arg `--https`, like in the current status of the PR. This way other people who might use punt37 doesn't have to worry about how to generate the SSL cert and configure it on the client side. We on our side can continue to use the `--https` flag and our cert workflow, which has proven to work with multiple testers.

Let me know how that sounds to you. Thanks.